pixterm: add page (#8411)
tldr-pages/tldr: pages/common/pixterm.md

# pixterm

> Image printing in the terminal.
> See also: `chafa`, `catimg`.
> More information: <https://github.com/eliukblau/pixterm>.

> - Print a static image to the terminal:

Reviewer: ```suggestion
- Print a static image:
```

Author: Similar suggestion by another reviewer on my `chafa` PR. I think it's important to mention that it's outputting to the terminal specifically, at least in the first example. Would this be a better clarification?

```suggestion
- Render a static image directly in the terminal:
```

Reviewer: 👍

`pixterm {{path/to/file}}`

- Use the image's original aspect ratio:

`pixterm -s 2 {{path/to/file}}`

> - Specify a custom aspect ratio with [t]erminal [r]ows and [c]olumns:

Reviewer: ```suggestion
- Specify a custom aspect ratio using a specific number of [t]erminal [r]ows and [c]olumns:
```

I think this makes more sense to anyone reading it, although it is a bit longer.
